src/sentry/web/urls.py

```diff
@@ -443,6 +443,7 @@ def init_all_applications():
     url(r'^crossdomain\.xml$', api.crossdomain_xml_index, name='sentry-api-crossdomain-xml-index'),
 
     # plugins
+    url(r'^integrations/cloudflare/', include('sentry.integrations.cloudflare.urls')),
```

Comment:
Oh wait, this won't work in our production since `/integrations/*` is mapped for static marketing pages. So this endpoint would never get hit by the app.